x-pack/plugins/task_manager/server/monitoring/task_run_statistics.ts

```diff
@@ -123,37 +126,57 @@ export function createTaskRunAggregator(
+  const taskPollingEvents$: Observable<Pick<TaskRunStat, 'polling'>> = combineLatest([
+    // get latest polling stats
+    taskPollingLifecycle.events.pipe(
+      filter(
+        (taskEvent: TaskLifecycleEvent) =>
+          isTaskPollingCycleEvent(taskEvent) &&
+          isOk<ClaimAndFillPoolResult, unknown>(taskEvent.event)
+      ),
+      map((taskEvent: TaskLifecycleEvent) => {
+        const {
+          result,
+          stats: { tasksClaimed, tasksUpdated, tasksConflicted } = {},
+        } = ((taskEvent.event as unknown) as Ok<ClaimAndFillPoolResult>).value;
+        const duration = (taskEvent?.timing?.stop ?? 0) - (taskEvent?.timing?.start ?? 0);
+        return {
+          polling: {
+            last_successful_poll: new Date().toISOString(),
+            // Track how long the polling cycle took from begining until all claimed tasks were marked as running
+            duration: duration ? pollingDurationQueue(duration) : pollingDurationQueue(),
+            // Track how many version conflicts occured during polling
+            claim_conflicts: isNumber(tasksConflicted)
+              ? claimConflictsQueue(tasksConflicted)
+              : claimConflictsQueue(),
+            // Track how much of a mismatch there is between claimed and updated
+            claim_mismatches:
+              isNumber(tasksClaimed) && isNumber(tasksUpdated)
+                ? claimMismatchesQueue(tasksUpdated - tasksClaimed)
+                : claimMismatchesQueue(),
+            result_frequency_percent_as_number: resultFrequencyQueue(result),
+          },
+        };
+      })
     ),
-    map((taskEvent: TaskLifecycleEvent) => {
-      const {
-        result,
-        stats: { tasksClaimed, tasksUpdated, tasksConflicted } = {},
-      } = ((taskEvent.event as unknown) as Ok<ClaimAndFillPoolResult>).value;
-      const duration = (taskEvent?.timing?.stop ?? 0) - (taskEvent?.timing?.start ?? 0);
-      return {
-        polling: {
-          last_successful_poll: new Date().toISOString(),
-          // Track how long the polling cycle took from begining until all claimed tasks were marked as running
-          duration: duration ? pollingDurationQueue(duration) : pollingDurationQueue(),
-          // Track how many version conflicts occured during polling
-          claim_conflicts: isNumber(tasksConflicted)
-            ? claimConflictsQueue(tasksConflicted)
-            : claimConflictsQueue(),
-          // Track how much of a mismatch there is between claimed and updated
-          claim_mismatches:
-            isNumber(tasksClaimed) && isNumber(tasksUpdated)
-              ? claimMismatchesQueue(tasksUpdated - tasksClaimed)
-              : claimMismatchesQueue(),
-          result_frequency_percent_as_number: resultFrequencyQueue(result),
-        },
-      };
-    })
+    // get DateTime of latest polling delay refresh
+    merge(
+      of(new Date().toISOString()),
```

Comment:
as `combineLatest` hangs until it has its first value and we're not likely to reconfigure the delay in normal deployments, we needed some initial value.
I've used _now_ (`new Date().toISOString()`) as it made the most sense (it would be the time Kibana started), but it _could_ be confusing in the future.
I might refactor this in the future to handle an empty `last_polling_delay` but due to the pressure to get this fix out for the next BC, I decided to put that off for now.

Reply:
**nit:** It could be worth moving this PR comment as a code comment for future reading.